src/main/resources/db/changeLog.yaml

```diff
@@ -2696,3 +2696,71 @@ databaseChangeLog:
+              UPDATE dataset SET license = 'gpl3' WHERE license = 'https://opensource.org/licenses/GPL-3.0';
+              UPDATE dataset_aud SET license = 'gpl3' WHERE license = 'https://opensource.org/licenses/GPL-3.0';
+              UPDATE dataset SET license = 'agpl1' WHERE license = 'http://www.affero.org/oagpl.html';
+              UPDATE dataset_aud SET license = 'agpl1' WHERE license = 'http://www.affero.org/oagpl.html';
+              UPDATE dataset SET license = 'agpl3' WHERE license = 'https://opensource.org/licenses/AGPL-3.0';
+              UPDATE dataset_aud SET license = 'agpl3' WHERE license = 'https://opensource.org/licenses/AGPL-3.0';
+              UPDATE dataset SET license = 'mpl2' WHERE license = 'https://opensource.org/licenses/MPL-2.0';
+              UPDATE dataset_aud SET license = 'mpl2' WHERE license = 'https://opensource.org/licenses/MPL-2.0';
+              UPDATE dataset SET license = 'lgpl21plus' WHERE license = 'https://opensource.org/licenses/LGPL-2.1';
+              UPDATE dataset_aud SET license = 'lgpl21plus' WHERE license = 'https://opensource.org/licenses/LGPL-2.1';
+              UPDATE dataset SET license = 'lgpl21' WHERE license = 'https://opensource.org/licenses/LGPL-2.1';
+              UPDATE dataset_aud SET license = 'lgpl21' WHERE license = 'https://opensource.org/licenses/LGPL-2.1';
+              UPDATE dataset SET license = 'lgpl3' WHERE license = 'https://opensource.org/licenses/LGPL-3.0';
+              UPDATE dataset_aud SET license = 'lgpl3' WHERE license = 'https://opensource.org/licenses/LGPL-3.0';
+              UPDATE dataset SET license = 'epl1' WHERE license = 'https://opensource.org/licenses/EPL-1.0';
+              UPDATE dataset_aud SET license = 'epl1' WHERE license = 'https://opensource.org/licenses/EPL-1.0';
+              UPDATE dataset SET license = 'cddl1' WHERE license = 'https://opensource.org/licenses/CDDL-1.0';
+              UPDATE dataset_aud SET license = 'cddl1' WHERE license = 'https://opensource.org/licenses/CDDL-1.0';
+              UPDATE dataset SET license = 'mit' WHERE license = 'https://opensource.org/licenses/mit-license.php';
+              UPDATE dataset_aud SET license = 'mit' WHERE license = 'https://opensource.org/licenses/mit-license.php';
+              UPDATE dataset SET license = 'bsd3c' WHERE license = 'https://opensource.org/licenses/BSD-3-Clause';
+              UPDATE dataset_aud SET license = 'bsd3c' WHERE license = 'https://opensource.org/licenses/BSD-3-Clause';
+              UPDATE dataset SET license = 'bsd2c' WHERE license = 'https://opensource.org/licenses/BSD-2-Clause';
+              UPDATE dataset_aud SET license = 'bsd2c' WHERE license = 'https://opensource.org/licenses/BSD-2-Clause';
+              UPDATE dataset SET license = 'apache2' WHERE license = 'https://www.apache.org/licenses/LICENSE-2.0';
+              UPDATE dataset_aud SET license = 'apache2' WHERE license = 'https://www.apache.org/licenses/LICENSE-2.0';
```

Comment:
are the old values the ones provided via the frontend (https://github.com/tuwien-csd/damap-frontend/pull/111/files#diff-bd187e92ade71b1f2f0ff7f6dbf17d4772981abeef78f1e3d7a0ba5db17797ddL1-L117)? If so, the script will have to check for `http://` instead of `https://`.

Also with regards to the comment above: might have to update the new names, if we go with all uppercase

Reply:
the old list with "http" was actually not used, therefore all entries in the databse are actually "https" values (found out the hard way)